extension/src/popup/views/AddAccount/ImportAccount/index.tsx

```diff
@@ -53,20 +55,23 @@ export const ImportAccount = () => {
 
   return (
     <div className="ImportAccount">
+      <SubviewHeader title="Import Stellar Secret Key" />
       <div className="ImportAccount__warning-block">
```

Comment:
Thanks for updating this view! I noticed it didn't have the header so I added that